satellite/marketing/marketingweb/server_test.go

```diff
@@ -0,0 +1,86 @@
+package marketingweb
```

Comment:
You can see an example of how to use `testplanet` for testing in `containment_test.go`

Reply:
pushed appending of `_test` to pkg name.

Reply:
Added test planet and followed proper http server unit test practices pushed